src/sampledata_registry.json

```diff
@@ -1,133 +1,133 @@
 {
   "Carbon Model": {
     "filename": "Carbon.zip",
-    "url": "https://www.googleapis.com/download/storage/v1/b/releases.naturalcapitalproject.org/o/invest%2F3.9.0.post285%2Bgf68dd440%2Fdata%2FCarbon.zip?generation=1614051202294534&alt=media",
-    "filesize": "1900216"
+    "url": "https://www.googleapis.com/download/storage/v1/b/releases.naturalcapitalproject.org/o/invest%2F3.9.0.post432%2Bgdecbeffb%2Fdata%2FCarbon.zip?generation=1622754766634004&alt=media",
+    "filesize": "1900623"
```

Comment:
Curious why we need to record the file size for all these?

Reply:
We're displaying it to the user in the download component, in units of MB. I think Doug reviewed that PR so you may not have seen that feature yet!